enterprise/backend/src/metabase_enterprise/llm/settings.clj

```diff
@@ -0,0 +1,15 @@
+(ns metabase-enterprise.llm.settings
+  (:require
+   [metabase.models.setting :refer [defsetting]]
+   [metabase.util.i18n :refer [deferred-tru]]))
+
+(defsetting ee-openai-model
+  (deferred-tru "The OpenAI Model (e.g. 'gpt-4', 'gpt-3.5-turbo')")
+  :visibility :settings-manager
+  :default "gpt-4-turbo-preview"
+  :export? false)
+
+(defsetting ee-openai-api-key
+  (deferred-tru "The OpenAI API Key used in Metabase Enterprise.")
+  :visibility :settings-manager
+  :export? false)
```

Comment:
I know this may not matter right now, but we do not have any key validation for this setting. An admin could mis-input a key, still have the feature show-up but not work correctly, and not know what went wrong cause everything else looks fine.

**Key**
![image](https://github.com/metabase/metabase/assets/22608765/7e617ca8-cb4e-4a83-a97c-bb3fa623bc4b)

**Clicking**
![image](https://github.com/metabase/metabase/assets/22608765/7af5315e-a906-4094-99da-56b8b373fd57)

**After click**
![image](https://github.com/metabase/metabase/assets/22608765/46ae86fb-8c8e-4cb6-941c-8212e3dca87d)

Reply:
Yeah, that will be in a follow-up.